src/routes.tsx

```diff
@@ -19,67 +14,31 @@ export function createRoutes( /*store*/ ): RouteConfig[] {
   // const { injectReducer, injectSagas } = getAsyncInjectors(store); // eslint-disable-line no-unused-vars
 
   return [
-    // {
-    //   // path: '/',
-    //   // name: 'home',
-    //   // component: Home
-    //
-    //   // getComponent( nextState, cb ) {
-    //   //   const importModules = Promise.all( [
-    //   //     System.import( './home/Home' ),
-    //   //   ] );
-    //   //
-    //   //   const renderRoute = loadModule( cb );
-    //   //
-    //   //   importModules.then( ( [component] ) => {
-    //   //     renderRoute( component );
-    //   //   } );
-    //   //
-    //   //   importModules.catch( errorLoading );
-    //   // },
-    // },
     {
-      path: 'about',
-      // name: 'about',
+      path: '*',
       getComponent( nextState, cb ) {
         const importModules = Promise.all( [
-          System.import( './about/About' ),
+          System.import( `./${nextState.location.pathname.slice(1)}/index` ),
```

Comment:
Btw this Promise.all is unnecessary in this case, we are loading just one file/module. Root one